ui/nuxeo-path-suggestion/nuxeo-path-suggestion.js

```diff
@@ -173,6 +174,24 @@ import { FormatBehavior } from '../nuxeo-format-behavior.js';
       };
     }
 
+    ready() {
+      super.ready();
```

Comment:
@semisse, we missed the `super.ready()` that is why the local DOM was not correctly initialized 🤦 Thank you @Gabez0r for spotting that

Reply:
Makes sense! I didn't see it either :)